xml/installation-networking-l2gw5930.xml

```diff
@@ -93,8 +93,7 @@
    the physical ports, the VLAN, and if it is an access or trunk port (tagged
    or untagged)
   </para>
-  <figure xml:id="fig1">
-   <title>Figure 1</title>
+  <informalfigure xml:id="fig1">
```

Comment:
Informal figures must not have IDs. (The reason for that is that the main reason we use IDs is to allow for xreffing, and since informalfigures have no title, the auto-generated xref would get pretty useless link text.)